EDDI/EDDI.cs

```diff
@@ -1864,14 +1937,16 @@ private bool eventEnteredNormalSpace(EnteredNormalSpaceEvent theEvent)
 
         private bool eventCrewJoined(CrewJoinedEvent theEvent)
         {
-            inCrew = true;
+            inTelepresence = true;
+            Vehicle = Constants.VEHICLE_MULTICREW;
             Logging.Info("Entering multicrew session");
             return true;
         }
 
         private bool eventCrewLeft(CrewLeftEvent theEvent)
         {
-            inCrew = false;
+            inTelepresence = false;
+            Vehicle = Constants.VEHICLE_SHIP;
```

Comment:
I'm not sure about this. What if I entered multicrew when on foot or in an SRV?

Reply:
I'm not sure the event fires for physical multicrew. Needs testing.

Reply:
Good question, need to check.

Reply:
I have confirmed that in Odyssey you can request multicrew from SRV and Suit, but I have not yet established what you return to.